internal/providers/pluginfw/resources/clusters/resource_library_test.go

```diff
@@ -0,0 +1,93 @@
+package clusters_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/databricks/terraform-provider-databricks/internal/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAccLibraryCreationPluginFramework(t *testing.T) {
+	acceptance.WorkspaceLevel(t, acceptance.Step{
+		Template: `data "databricks_spark_version" "latest" {
+		}
+		resource "databricks_cluster" "this" {
+			cluster_name = "libs-{var.RANDOM}"
+			spark_version = data.databricks_spark_version.latest.id
+			instance_pool_id = "{env.TEST_INSTANCE_POOL_ID}"
+			autotermination_minutes = 10
+			num_workers = 1
+			spark_conf = {
+				"spark.databricks.repl.allowedLanguages" = "sql,python,r"
+				"spark.databricks.cluster.profile" = "serverless"
+			}
+			custom_tags = {
+				"ResourceClass" = "Serverless"
+			}
+		}
+		resource "databricks_library_pluginframework" "new_library" {
+			cluster_id = databricks_cluster.this.id
+			pypi = {
+				repo = "https://pypi.org/simple"
+				package = "databricks-sdk"
+			}
+		}
+		`,
+	})
+}
+
+func TestAccLibraryImportPluginFramework(t *testing.T) {
+	acceptance.WorkspaceLevel(t,
+		acceptance.Step{
+			Template: `data "databricks_spark_version" "latest" {
+		}
+		resource "databricks_cluster" "this" {
+			cluster_name = "libs-{var.RANDOM}"
+			spark_version = data.databricks_spark_version.latest.id
+			instance_pool_id = "{env.TEST_INSTANCE_POOL_ID}"
+			autotermination_minutes = 10
+			num_workers = 1
```

Comment:
Do we need a worker to test library installation? It will be half as expensive to run this test this way.

Reply:
we can update it to 0 and use a single node cluster